src/status_im/ui/components/toolbar/styles.cljs

```diff
@@ -7,70 +7,38 @@
 (def toolbar-icon-height 24)
 (def toolbar-icon-spacing 24)
 
-(defnstyle toolbar [background-color flat?]
-  (cond->
-   {:flex            0
-    :flex-direction  :row
-    :align-items     :center
-    :justify-content :space-between
-    :elevation       (if flat? 0 2)
-    :android         {:height 55}
-    :ios             {:height 56}}
-
-    background-color
-    (assoc :background-color background-color)))
-
-(def toolbar-nav-actions-container
-  {:flex-direction :row
-   :margin-left    4})
-
-(defstyle toolbar-container
-  {:flex        1
-   :align-items :center})
+(def toolbar
+  {:height 55
+   :flex   1})
 
 (def toolbar-title-container
-  {:flex           1
-   :flex-direction :column
-   :margin-left    6})
+  {:justify-content :center
+   :align-items     :center
+   :flex-direction  :column
+   :margin-left     6})
 
 (defstyle toolbar-title-text
   {:color          colors/black
    :letter-spacing -0.2
    :font-size      17
    :font-weight    :bold
-   :ios            {:text-align     :center
-                    :margin-left    15
-                    :margin-right   15}
-   :android        {:text-align     :left
-                    :margin-left    22}})
-
-(def toolbar-actions
-  {:flex           0
-   :flex-direction :row})
+   :text-align     :center
+   :margin-left    15
+   :margin-right   15})
```

Comment:
shouldn't it be 16? I've noticed it's 15 everywhere, are we subtracting the -1 for something?